.github/ISSUE_TEMPLATE/01_feature_request.yml

```diff
@@ -0,0 +1,31 @@
+name: 🚀 Feature Request
```

Comment:
skąd to tutaj?

Reply:
Dobre pytanie. Wyczyszę

Reply:
Właśnie sprawdziłem i dodał to @mhordynski. Nie wiem na ile mogę to usuwać